scene/gui/tree.h

```diff
@@ -524,6 +527,11 @@ class Tree : public Control {
 	}
 
 public:
+  
+  	/* Test for focus loss */
+	void set_dragExited(bool b);
```

Comment:
You should use meaningful parameter names, one letter `b` doesn't say anything about its purpose.
As you can see in other methods of this class and all over the Godot API, parameters should be prefixed with `p_`.

But actually, I don't see the point of those public methods. `drag_exited` only needs to be a private bool, you can change it directly from within the class. The setter and getter don't add any value.